### PR TITLE
refactor: update test cases for reworked bulk validation

### PR DESCRIPTION
**Pull request by kleyow**
Update bulk test cases to error codes found in sequence diagram https://docs.mojaloop.io/legacy/mojaloop-technical-overview/central-bulk-transfers/transfers/1.1.1-bulk-prepare-handler-consume.html

### CODE REVIEW

File: collections/hub/other_tests/bulk_transfers/bulk-invalid-dfsp-name-header-and-body.json [hunk 1]
```diff
@@ -201,10 +201,129 @@
...
+          "params": {
+            "name": "testingtoolkitdfsp",
+            "ID": ""
+          },
+          "path": "/bulkTransfers",
+          "scriptingEngine": "javascript",
+          "scripts": {
+            "preRequest": {
+              "exec": [
+                "const delay = 1000 * 3600 * 24",
```
Comment (mdebarros) [inline collections/hub/other_tests/bulk_transfers/bulk-invalid-dfsp-name-header-and-body.json:222]: Can we parametrise this please, and revert to the above default if said parameter is not set?

I.e. something like `const delay = environment.DELAY_THING_PARAM || 1000 * 3600 * 24`

Comment (mdebarros) [reply collections/hub/other_tests/bulk_transfers/bulk-invalid-dfsp-name-header-and-body.json:222]: I found an example --> https://github.com/mojaloop/testing-toolkit-test-cases/blob/4d6c44aad47da64a4e68d0e49c84929d05bb5edb/collections/hub/golden_path/settlement_cgs/newsetcgs.json#L1554

Comment (kleyow) [reply collections/hub/other_tests/bulk_transfers/bulk-invalid-dfsp-name-header-and-body.json]: done.

**Review by mdebarros — APPROVED** [review]
+1

**Review by elnyry-sam-k — APPROVED** [review]

**Pull request merged**